frontend/src/general/simpleTable/SimpleTableUtil.js

```diff
@@ -0,0 +1,8 @@
+export function zipProperties(propertyNames, properties) {
```

Comment:
Get rid of the code smell if u can (prefer default export). Or move this code to be in a free function in SimpleTable.jsx.

Reply:
put it in simpletable as u suggested!